### DISCUSSION TITLE
Does `StreamSink<ZeroCopyBuffer<Vec<u8>>>` move the data instead of copying it?

### DISCUSSION THREAD

**Discussion by temeddix** [fzyzcjy/flutter_rust_bridge#1226] [Q&A] [answered]
I know this is a rather simple question, but I just wanted to make it sure. Does `StreamSink` use `allo-isolate` internally to convert Rust objects into Dart?

**Comment by fzyzcjy** [comment, marked as answer]
IIRC yes, since StreamSink does nothing but send data from Rust to Dart in the same way as normal return. But I would suggest making simple experiments to check :)
